app/uk/gov/hmrc/hec/models/hecTaxCheck/CTAccountingPeriod.scala

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package uk.gov.hmrc.hec.models
+package uk.gov.hmrc.hec.models.hecTaxCheck
```

Comment:
I wonder if it'd be worth splitting the company and individual stuff into separate subpackages as well, where things like
```
CTAccountingPeriod
CTStatus
CTStatusResponse`
CompanyHouseName
```
would go under the company one and
```
DateOfBirth
Name
SAStatus
SAStatusResponse
```
would go in the `individual` one. We've also got a `licence` package under `models` which might sit better being moved under `hecTaxCheck` since it's all stuff to do with the tax check data

Reply:
updated https://github.com/hmrc/hec/pull/54/commits/36cd5564fe324c32f69a15a805ff94fde428d958